apps/blockchain/test/blockchain_test.exs

```diff
@@ -1,4 +1,1486 @@
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d114g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d115g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d116g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d117g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d118g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d119g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d11g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d120g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d121g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d122g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d123g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d124g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d125g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d126g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d12g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d13g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d14g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d15g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d16g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d17g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d18g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d19g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d1g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d20g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d21g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d22g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d23g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d24g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d25g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d26g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d27g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d28g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d29g0v0.json",
+    # "./GeneralStateTests/stTransactionTest/Opcodes_TransactionInit_d2g0v0.json",
+    # "./Gene
```

Comment:
Could some of the functions in `EthCommonTest.Helpers` be helpful here?

Reply:
I think these functions will be useful when we will fix all blockchain tests. Currently, it's very helpful to see which tests are failing